src/main/java/io/fabric8/maven/docker/service/BuildXService.java

```diff
@@ -245,27 +245,31 @@ protected void createDirectory(Path cachePath) {
     protected String createBuilder(Path configPath, List<String> buildX, ImageConfiguration imageConfig, BuildDirs buildDirs) throws MojoExecutionException {
         BuildXConfiguration buildXConfiguration = imageConfig.getBuildConfiguration().getBuildX();
         String builderName = Optional.ofNullable(buildXConfiguration.getBuilderName()).orElse("maven");
-        String nodeName = buildXConfiguration.getNodeName();
-        Path builderPath = configPath.resolve(Paths.get("buildx", "instances", builderName));
-        if(Files.notExists(builderPath)) {
-            List<String> cmds = new ArrayList<>(buildX);
-            append(cmds, "create", "--driver", "docker-container", "--name", builderName);
-            if (nodeName != null) {
-                append(cmds, "--node", nodeName);
-            }
```

Comment:
Docker does a nice job of providing meaningful error messages if you try to use the default builder with multiple architectures, etc.  For instance:
```
[INFO] DOCKER> ERROR: Multi-platform build is not supported for the docker driver.
[INFO] DOCKER> Switch to a different driver, or turn on the containerd image store, and try again.
[INFO] DOCKER> Learn more at https://docs.docker.com/go/build-multi-platform/
[ERROR] DOCKER> Error status (1) when building
```